ext/oboe_metal/README.md

```diff
@@ -65,3 +65,88 @@ Some inspiring examples here:
 <https://jvns.ca/blog/2016/06/12/a-weird-system-call-process-vm-readv/>
 
 <https://medium.com/@zanker/finding-a-ruby-bug-with-gdb-56d6b321bc86>
+
+## Debug the c-code with core dump
```

Comment:
How about making this section the same heading level as `# Debug the c-code with gdb`?

And just realized, maybe step 1 (install solarwinds with debug) should go after 2 and 3, i.e. make sure coredump file can be gathered first, then use special install to try reproduce crash.

Reply:
Updated both.